File: templates/python/microgrid_market_scenario_facets.py
"""microgrid_market_scenario_facets: 微电网与市场场景分面（small-multiples 模式，合成数据）."""
import sys, os
sys.path.insert(0, os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', '..', '_utils', 'python'))
from generated_patterns import make_template_figure


def make_figure(title='microgrid and market analysis: scenario small multiples'):
    return make_template_figure('small_multiples', seed=3809, title=title, domain='microgrid and market analysis', topic='scenario small multiples')


if __name__ == '__main__':
    fig = make_figure()
    fig.savefig(__file__.replace('.py', '.png'), dpi=150)

File: templates/matlab/microgrid_market_scenario_facets.m
function fig = microgrid_market_scenario_facets()
    addpath(fullfile(fileparts(mfilename('fullpath')),'..','..','_utils','matlab'));
    fig = generated_template_figure('small_multiples', 3809, 'microgrid and market analysis: scenario small multiples', 'microgrid and market analysis', 'scenario small multiples');
end
